<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Shop_dialog</class>
 <widget class="QDialog" name="Shop_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="n_b">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>220</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color: rgb(255, 110, 43);</string>
   </property>
   <property name="text">
    <string>buy</string>
   </property>
  </widget>
  <widget class="QLabel" name="nail_l">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>110</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="n_s">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>220</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color: rgb(255, 110, 43);</string>
   </property>
   <property name="text">
    <string>sell</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bil_b">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>220</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color: rgb(255, 110, 43);</string>
   </property>
   <property name="text">
    <string>buy</string>
   </property>
  </widget>
  <widget class="QLabel" name="bil_l">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>110</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bil_s">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>220</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color: rgb(255, 110, 43);</string>
   </property>
   <property name="text">
    <string>sell</string>
   </property>
  </widget>
  <widget class="QLabel" name="milk_l">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="milk_s">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>220</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color: rgb(255, 110, 43);</string>
   </property>
   <property name="text">
    <string>sell</string>
   </property>
  </widget>
  <widget class="QPushButton" name="g_b">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>220</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color: rgb(255, 110, 43);</string>
   </property>
   <property name="text">
    <string>buy</string>
   </property>
  </widget>
  <widget class="QLabel" name="gandom_l">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="g_s">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color: rgb(255, 110, 43);</string>
   </property>
   <property name="text">
    <string>sell</string>
   </property>
  </widget>
  <widget class="QPushButton" name="y_b">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color: rgb(255, 110, 43);</string>
   </property>
   <property name="text">
    <string>buy</string>
   </property>
  </widget>
  <widget class="QLabel" name="yonje_l">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="y_s">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color: rgb(255, 110, 43);</string>
   </property>
   <property name="text">
    <string>sell</string>
   </property>
  </widget>
  <widget class="QLabel" name="pashm_l">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="p_s">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>410</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color: rgb(255, 110, 43);</string>
   </property>
   <property name="text">
    <string>sell</string>
   </property>
  </widget>
  <widget class="QPushButton" name="s_b">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>410</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color: rgb(255, 110, 43);</string>
   </property>
   <property name="text">
    <string>buy</string>
   </property>
  </widget>
  <widget class="QLabel" name="sheep_l">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="s_s">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>410</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color: rgb(255, 110, 43);</string>
   </property>
   <property name="text">
    <string>sell</string>
   </property>
  </widget>
  <widget class="QPushButton" name="c_b">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>410</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color: rgb(255, 110, 43);</string>
   </property>
   <property name="text">
    <string>buy</string>
   </property>
  </widget>
  <widget class="QLabel" name="cow_l">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>300</y>
     <width>131</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="c_s">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>410</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color: rgb(255, 110, 43);</string>
   </property>
   <property name="text">
    <string>sell</string>
   </property>
  </widget>
  <widget class="QPushButton" name="h_b">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>410</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color: rgb(255, 110, 43);</string>
   </property>
   <property name="text">
    <string>buy</string>
   </property>
  </widget>
  <widget class="QLabel" name="hen_l">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>300</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="h_s">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>410</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color: rgb(255, 110, 43);</string>
   </property>
   <property name="text">
    <string>sell</string>
   </property>
  </widget>
  <widget class="QLabel" name="egg_l">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>300</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="e_s">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>410</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color: rgb(255, 110, 43);</string>
   </property>
   <property name="text">
    <string>sell</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_n">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>270</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(172, 255, 17);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_b">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>270</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(172, 255, 17);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_m">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>270</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(172, 255, 17);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_g">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>270</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(172, 255, 17);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_y">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>270</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(172, 255, 17);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_p">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>460</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(172, 255, 17);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_s">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>460</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(172, 255, 17);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_c">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>460</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(172, 255, 17);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_h">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>460</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(172, 255, 17);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_e">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>460</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(172, 255, 17);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nn">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>100</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nb">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>100</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nm">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ng">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ny">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ne">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>290</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nh">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>290</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nc">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>290</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ns">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>290</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_np">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="edit_l">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>-10</y>
     <width>251</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="edit2_l">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>281</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>n_b</zorder>
  <zorder>nail_l</zorder>
  <zorder>n_s</zorder>
  <zorder>bil_b</zorder>
  <zorder>bil_l</zorder>
  <zorder>bil_s</zorder>
  <zorder>milk_l</zorder>
  <zorder>milk_s</zorder>
  <zorder>g_b</zorder>
  <zorder>gandom_l</zorder>
  <zorder>g_s</zorder>
  <zorder>y_b</zorder>
  <zorder>yonje_l</zorder>
  <zorder>pashm_l</zorder>
  <zorder>p_s</zorder>
  <zorder>s_b</zorder>
  <zorder>sheep_l</zorder>
  <zorder>s_s</zorder>
  <zorder>c_b</zorder>
  <zorder>cow_l</zorder>
  <zorder>c_s</zorder>
  <zorder>h_b</zorder>
  <zorder>hen_l</zorder>
  <zorder>h_s</zorder>
  <zorder>egg_l</zorder>
  <zorder>e_s</zorder>
  <zorder>lineEdit_n</zorder>
  <zorder>lineEdit_b</zorder>
  <zorder>lineEdit_m</zorder>
  <zorder>lineEdit_g</zorder>
  <zorder>lineEdit_y</zorder>
  <zorder>lineEdit_p</zorder>
  <zorder>lineEdit_s</zorder>
  <zorder>lineEdit_c</zorder>
  <zorder>lineEdit_h</zorder>
  <zorder>lineEdit_e</zorder>
  <zorder>label_nn</zorder>
  <zorder>label_nb</zorder>
  <zorder>label_nm</zorder>
  <zorder>label_ng</zorder>
  <zorder>label_ny</zorder>
  <zorder>label_ne</zorder>
  <zorder>label_nh</zorder>
  <zorder>label_nc</zorder>
  <zorder>label_ns</zorder>
  <zorder>label_np</zorder>
  <zorder>edit_l</zorder>
  <zorder>edit2_l</zorder>
  <zorder>y_s</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
